### ISSUE TITLE
Game crashes during catching tutorial

### ISSUE THREAD

**Issue by SpecterSpectrum** [mFireworks/Triple-Battle-Converter#4]
When setting up the game (Pokemon White 2), everything seemed to be fine. I even used the BWtool to make sure that every battle was a triple battle. Though I suppose that isn't the problem here. The first rival battle went just fine, but once I got to the catching tutorial, the game froze right before Bianca encountered a pokemon. I tried doing the whole process again with a new seed, but it still crashed.

**Comment by mFireworks** [comment]
Just now seeing this, people on youtube have been commenting about this issue as well. What's strange about it is that I've never run into this issue in all the testing that I've done, so it's hard to reproduce. So far, I know the following:
- Really unlikely it's issue with the triple battles mod, as I don't think the catching tutorial is a trainer battle.
- An issue with exporting the files, at it appears you've verified your trainer battles just fine, so wasn't an improper extraction.

That leaves me to believe it has to do with some settings in PokeRando, it's the only other thing that modifies the game. In PokeRando, there's a `Save Settings` button that'll create a rnqs file. Can you export the settings you used and upload them here so I can replicate that and see if I get the same crash?

**Comment by SpecterSpectrum** [comment]
> That leaves me to believe it has to do with some settings in PokeRando, it's the only other thing that modifies the game. In PokeRando, there's a `Save Settings` button that'll create a rnqs file. Can you export the settings you used and upload them here so I can replicate that and see if I get the same crash?

This? [randomizersettings.zip](https://github.com/mFireworks/Triple-Battle-Converter/files/14527667/randomizersettings.zip)
I didn't put it through the editors if that matters.

**Comment by mFireworks** [comment]
Yeah, that looks like the right file. I'll use those settings and see if I can reproduce the crash tonight.

**Comment by SpecterSpectrum** [comment]
I should mention that i use desmume, just in case that matters.

**Comment by mFireworks** [comment]
@SpecterSpectrum so I was able to reproduce the crash on the catching tutorial with your settings. Didn't even use the triple battle mod so it definitely caused by the randomizer. Looking that PokeRando's github, [I found this exact issue logged](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/issues/654).

Turns out, just turn off `Fix Music` under Static Pokemon in the Starters, Statics, and Trades tab and it should work just fine. Closing this issue as it doesn't require a change here. Will upload a new version of the mod today with the fix for #3.
